subprojects/docs/src/docs/userguide/migrating_from_maven.adoc

```diff
@@ -77,50 +85,200 @@ $ gradle build
+Don't worry though, you have a lot of control over which versions are selected, as documented in <<managing_transitive_dependencies#managing_transitive_dependencies,Managing Transitive Dependencies>>.
 
-Gradle has supported the importing of BOMs since version 4.6, although you must enable this explicitly for versions prior to 5.0 by adding the following to your settings script:
+Over the following sections, we will show you how to migrate the most common elements of a Maven build's dependency management information.
 
-.settings.gradle
+[[migmvn:declaring_deps]]
+=== Declaring dependencies
+
+Gradle uses the same dependency identifier components as Maven: group ID, artifact ID and version.
+It also supports classifiers.
+So all you need to do is substitute the identifier information for a dependency into Gradle's syntax, which is described in the <<declaring_dependencies#declaring_dependencies,Declaring Dependencies>> chapter.
+
+For example, consider this Maven-style dependency on Log4J:
+
+[source,xml]
+----
+<dependencies>
+    <dependency>
+        <groupId>log4j</groupId>
+        <artifactId>log4j</artifactId>
+        <version>1.2.12</version>
+    </dependency>
+</dependencies>
+----
+
+This dependency would look like the following in a Gradle build script:
+
+[source,groovy]
+----
+dependencies {
+    implementation 'log4j:log4j:1.2.12'  // <1>
+}
+----
+<1> Attaches version 1.2.12 of Log4J to the `implementation` configuration (scope)
+
+The string identifier takes the form "__<groupId>__:__<artifactId>__:__<version>__", although Gradle refers to them as "group", "module" and "version".
+
+The above example raises an obvious question: what is that `implementation` configuration?
+It's one of the standard dependency configurations provided by the <<java_plugin#tab:configurations,Java Plugin>> and is often used as a substitute for Maven's default `compile` scope.
+
+Several of the differences between Maven's scopes and Gradle's standard configurations come down to Gradle distinguishing between the dependencies required to build a module and the dependencies required to build a module that depends on it.
+Maven makes no such distinction, so published POMs typically include dependencies that consumers of a library don't actually need.
+
+Here are the main Maven dependency scopes and how you should deal with their migration:
+
+`compile`::
+Gradle has two configurations that can be used in place of the `compile` scope: `implementation` and `api`.
+The former is available to any project that applies the Java Plugin, while `api` is only available to projects that specifically apply the <<java_library_plugin#java_library_plugin,Java Library Plugin>>.
+
++
+In most cases you should simply use the `implementation` configuration, but read the section on <<building_java_projects#sec:building_java_libraries,Building Java libraries>> and the chapter on the Java Library Plugin to find out whether you should use `api` instead for a given dependency.
```

Comment:
Suggestion to clarify:

`..., but read the section...` ->
`..., in particular if you are building an application. If you are building a library, or split your project into several subprojects, you should consult the section...`